guides/common/assembly_introduction-to-project-ansible-module-collection.adoc

```diff
@@ -0,0 +1,3 @@
+include::modules/con_introduction-to-project-ansible-module-collection.adoc[]
+
+include::modules/ref_getting-help-with-project-ansible-module-collection.adoc[leveloffset=+1]
```

Comment:
getting help uses the built in docs, but in some cases the collection might not be installed yet. is that OK for the introduction?

Reply:
Hm, I may have copied the Hammer guide's structure too blindly. Updated so that users first install the package.